x/blob/types/payforblob_test.go

```diff
@@ -110,6 +111,10 @@ func TestCreateCommitment(t *testing.T) {
 	}
 }
 
+func TestMsgTypeURLParity(t *testing.T) {
+	require.Equal(t, sdk.MsgTypeURL(&MsgPayForBlobs{}), URLMsgPayForBlobs)
+}
```

Comment:
Added a little test to make sure these two values are in parity. I think in v0.47 of the SDK we no longer will need to do this.